translations/en.json

```diff
@@ -625,7 +625,8 @@
     "follow-your-interests": "Jump into a public chat and meet new people",
     "follow": "Follow",
     "free": "↓ Free",
-    "from": "From",
+    "From": "From",
```

Comment:
Definitely not a strong opinion, but IMO is better to maintain one lowercase string and capitalize dynamically.

Reply:
I'm not sure you actually want that. I imagine capitalising in different languages can have different/ unexpected effects and I think it's best to leave to the translation string to handle that. Could be wrong on this ofc 🤷‍♂️

Reply:
Given that the translation keys end up being keywords in our codebase `:t/<something>`, I think we should just consider the idiomatic Clojure code to decide what to do. In Clojure, keywords are case sensitive, but I've never seen any usage of keywords with mixed case, therefore they should be lower case.

Reply:
Also didn't like that approach, but we already used it in few times in translations.

But Instead of using capitalized keywords or hoping that `(capitalize)` won't affect translation, we can use keywords like `:t/from-capitalized`. I believe they will be obvious enough)